packages/simulator/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "@tsconfig/node16-strictest-esm/tsconfig.json",
   "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
```

Comment:
Why do we need this changes ?

Reply:
Was necessary to "execute" a ts file with ts-node.